server/services/CsvGeneratorService v2.js

```diff
@@ -0,0 +1,787 @@
+/*
```

Comment:
is this code copy from elastic?

the file name of this js file confused me?
why add whitespace in file name ```CsvGeneratorService v2.js```